ui/plugins/controller.js

```diff
@@ -397,14 +397,15 @@ treeherder.controller('PluginCtrl', [
                                 const backfilltask = _.find(results.actions, {name: 'backfill'});
                                 // We'll fall back to actions.yaml if this isn't true
                                 if (backfilltask) {
-                                    const actionTask = tcactions.render(backfilltask.task, _.defaults({}, {
-                                        taskGroupId: decisionTaskId,
+                                    return tcactions.submit({
```

Comment:
This no longer uses `queue` so you can remove the vars above, right (and for other actions too)?  Or is it still used for the actions.yml fallback?